hw/drivers/bq27z561/src/bq27z561.c

```diff
@@ -663,6 +663,7 @@ bq27z561_battery_property_get(struct battery_driver *driver,
         rc = -1;
         assert(0);
     }
+    property->bp_valid = rc == 0;
```

Comment:
I personally do not like "cryptic" lines of code like this. Makes people think too hard. There are other ways to write that line of code that would not force anyone to think about it :-)

Reply:
i agree.

Reply:
I will change accordingly it and merge if there no more doubts.